nhais-adaptor/Pipfile.lock

```diff
@@ -1,7 +1,11 @@
 {
     "_meta": {
         "hash": {
+<<<<<<< HEAD
```

Comment:
Looks like these are merge conflicts. They'll need fixing (presumably there shouldn't be any changes to the NHAIS adaptor Pipfiles as part of this PR?).

Reply:
Good spot, think this must have appeared when I was running Fawad's tests, I'll update it to match master